packages/react-notion-x/src/third-party/collection-view-calendar.tsx

```diff
@@ -0,0 +1,436 @@
+                            <div className='notion-calendar-body-inner-card-inner-box-title'>
+                              <div className='notion-calendar-body-inner-card-inner-box-title-inner'>
+                                <div className='notion-calendar-body-inner-card-inner-box-title-inner-icon'>
+                                  <DefaultPageIcon className='notion-calendar-body-inner-card-inner-box-title-inner-icon-svg' />
+                                </div>
+                                <div className='notion-calendar-body-inner-card-inner-box-title-inner-text'>
+                                  <Property
+                                    schema={titleSchema}
+                                    data={titleData}
+                                    block={block}
+                                    collection={collection}
+                                    linkToTitlePage={false}
+                                  />
+                                </div>
+                              </div>
+                            </div>
+                            <div className='notion-calendar-body-inner-card-inner-box-properties'>
+                              {collectionView.format?.calendar_properties
+                                ?.filter((p) => p.visible)
+                                .map((p, z) => {
+                                  const schema = collection.schema[p.property]
+                                  const data =
+                                    block && block.properties?.[p.property]
+
+                                  if (!schema) {
+                                    return null
+                                  }
+
+                                  return (
+                                    <div
+                                      className='notion-calendar-body-inner-card-inner-box-properties-property'
+                                      key={z}
+                                    >
+                                      <Property
+                                        schema={schema}
+                                        data={data}
+                                        block={block}
+                                        collection={collection}
+                                        longMonth={true}
+                                      />
+                                    </div>
+                                  )
+                                })}
+                            </div>
+                            <div
+                              style={{
+                                position: 'absolute',
+                                top: '0px',
+                                left: '-4px',
+                                height: '100%',
+                                width: '12px',
+                                cursor: 'col-resize'
+                              }}
+                            ></div>
+                            <div
```

Comment:
Just want to understand; why are there two of the same absolute `div` here? They seem to be identical?